### PR TITLE
Migrate `instanceof` to `is`

### PR DESCRIPTION
**Pull request by lexidor**

### CODE REVIEW

File: src/ExpectObj.hack [hunk 1]
```diff
@@ -327,12 +327,13 @@ class ExpectObj<T> extends Assert {
...
     $actual = $this->var;
     invariant(
-      $actual instanceof Traversable,
+      $actual is Traversable<_>,
       'ERROR: expect(...$args)->toBeSortedByKey only can be applied to '.
       'Traversables, not %s.',
       print_type($actual),
     );
 
+    /*HH_FIXME[4110] I am uncertain about this error.*/
```
Comment (lexidor) [inline src/ExpectObj.hack:336]: This seems like it shouldn't work. How is Tv enforced?

**Comment by lexidor** [comment]
Addresses https://github.com/hhvm/fbexpect/issues/12

**Comment by lexidor** [comment]
As for travis. HHVM4.1 is angry about vendor mostly. I am not quite sure what it is trying to tell us...

File: .hhconfig [hunk 2]
```diff
@@ -6,3 +6,6 @@ disallow_unsafe_comparisons=true
 decl_override_require_hint=true
 enable_experimental_tc_features=shape_field_check,sealed_classes
 user_attributes=
+disable_unsafe_expr = true
+disable_unsafe_block = true
+; disable_instanceof = true
```
Comment (azjezz) [inline .hhconfig]: new line

**Comment by fredemmott** [comment]
Sorry, unable to take this given the current performance of `is Foo` with generics without repo-auth mode.

**Pull request closed without merging**